dev/io.openliberty.org.apache.myfaces.3.0/bnd.bnd

```diff
@@ -0,0 +1,128 @@
+  org.apache.myfaces.push.cdi;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.renderkit;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.shared.context.flash;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.shared.taglib;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.shared.taglib.core;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.shared.util;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.spi.impl;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.taglib.core;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.taglib.html;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view.facelets;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view.facelets.compiler;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view.facelets.component;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view.facelets.el;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view.facelets.impl;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view.facelets.pool;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view.facelets.pool.impl;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view.facelets.tag;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view.facelets.tag.composite;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view.facelets.tag.jsf;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view.facelets.tag.jsf.core;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view.facelets.tag.jsf.html;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view.facelets.tag.jstl.core;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view.facelets.tag.jstl.fn;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view.facelets.tag.ui;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.view.facelets.util;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.webapp;thread-context=true;version=3.0.0, \
+  org.apache.myfaces.*;version=3.0.0
+
+# Import everything we need except for the below exclusions that will not be needed at runtime.
+Import-Package: \
+  !com.ibm.ws.jsf.ee, \
+  !com.google.inject, \
+  !jakarta.ejb, \
+  !jakarta.persistence, \
+  !org.apache.tomcat, \
+  *
+
+-buildpath: \
+  org.apache.myfaces.core:myfaces-impl;version=3.0.0,\
+  io.openliberty.jakarta.faces.3.0;version=latest,\
+  com.ibm.ws.classloading;version=latest,\
+  com.ibm.ws.webcontainer;version=latest,\
+  com.ibm.ws.serialization;version=latest,\
+  com.ibm.ws.container.service;version=latest,\
+  com.ibm.ws.adaptable.module;version=latest,\
+  com.ibm.ws.anno;version=latest,\
+  com.ibm.ws.managedobject;version=latest,\
+  com.ibm.ws.jsp;version=latest,\
+  io.openliberty.jakarta.el.4.0;version=latest,\
+	io.openliberty.jakarta.servlet.5.0;version=latest, \
```

Comment:
Can you clean up the formatting here as well?

Reply:
will do